app/controllers/posts_controller.rb

```diff
@@ -32,6 +32,7 @@ def create
   def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
+    @comments = @post.comments.includes(:author)
```

Comment:
- [ ] [OPTIONAL] We can do the same in the index action with this code: __@posts = @user.posts.includes(:comments)__